packages/webapp/src/components/Finances/PureFinanceTypeSelection/CantFindCustomType/index.jsx

```diff
@@ -0,0 +1,94 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation, Trans } from 'react-i18next';
+import styles from './styles.module.scss';
+import { IconLink } from '../../../Typography';
+import { ReactComponent as Edit } from '../../../../assets/images/edit.svg';
+import { ReactComponent as UncheckedEnabled } from '../../../../assets/images/unchecked-enabled.svg';
+import { ReactComponent as CheckedEnabled } from '../../../../assets/images/checked-enabled.svg';
+
+export const CantFindCustomType = ({
+  customTypeMessages,
+  miscellaneousConfig,
+  iconLinkId,
+  onGoToManageCustomType,
+}) => {
+  const { t } = useTranslation();
+
+  const isChecked = miscellaneousConfig?.selected;
+
+  const handleCheck = () => {
+    miscellaneousConfig?.addRemove();
+  };
+
+  return (
+    <div className={styles.background}>
+      <div className={styles.container}>
+        <div className={styles.textContainer}>
+          <h3 className={styles.title}>{t('FINANCES.CANT_FIND.MAIN')}</h3>
+          <p className={styles.subtitle}>{customTypeMessages.info}</p>
+        </div>
+
+        <div className={styles.linkContainer}>
+          <IconLink
+            className={styles.manageCustomLink}
+            id={iconLinkId}
+            icon={<Edit />}
+            onClick={onGoToManageCustomType}
+            isIconClickable
+            underlined
+          >
+            {customTypeMessages.manage}
+          </IconLink>
+
+          {miscellaneousConfig && (
+            <div className={styles.miscellaneous} onClick={handleCheck}>
```

Comment:
Since this has a click effect, it should actually be a button for accessibility purposed. I believe @SayakaOno recently created a `TextButton` component for this purpose that you could use here

Reply:
Should it be a button? Or should these tiles be "inputs" since they are checkboxes to populate a form?

Reply:
Sorry I wrote it below but it belonged here; they are only fake checkboxes and not part of a real form!

And `<TextButton />` is quite cool and flexible! Neat!